kfs-core/src/main/java/edu/arizona/kfs/sys/identity/ArizonaKfsKimAttributes.java

```diff
@@ -0,0 +1,58 @@
+/*
```

Comment:
Please rename module KfsKimAttributes. We are trying to do away with the "AZ" or "Arizona" name prefaces. Also, please remove license text as this is the old license. When and if we contribute this, we will add in the most current license text.